internal/distro/rhel84/distro.go

```diff
@@ -547,11 +543,22 @@ func (t *imageType) grub2StageOptions(pt *disk.PartitionTable, kernelOptions str
 		legacy = t.arch.legacy
 	}
 
+	var kernelEntry string
+	if kernel != nil {
+		kernelOptions += " " + kernel.Append
+		for _, pkg := range packages {
+			if pkg.Name == kernel.Name {
+				kernelEntry = "ffffffffffffffffffffffffffffffff-" + pkg.Version + "-" + pkg.Release + "." + pkg.Arch + ".conf"
```

Comment:
I think it is without the `.conf` at the end:
```
at /boot/grub2/grubenv 
# GRUB Environment Block
kernelopts=root=UUID=0194fdc2-fa2f-4cc0-81d3-ff12045b73c8 console=tty0 console=ttyS0,115200n8 no_timer_check net.ifnames=0 crashkernel=auto
saved_entry=ffffffffffffffffffffffffffffffff-4.18.0-275.el8.x86_64
```

Reply:
Thanks!